deploy/docker/docker-compose.yml

```diff
@@ -19,6 +19,9 @@ version: "3.8"
 services:
   dolphinscheduler-postgresql:
     image: bitnami/postgresql:11.11.0
```

Comment:
I would suggest using the official image of postgres instead of a third party image. Which is more stablity and more authority.

Reply:
> I would suggest using the official image of postgres instead of a third party image. Which is more stablity and more authority.

Do you mean `https://hub.docker.com/_/postgres?tab=description`? Can you give some suggestion of the stable version?

Reply:
Yes. We use this image as datasource E2E test in ```dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/datasource-postgresql/docker-compose.yaml```. But I'm not familiar with postgres stable version...

Reply:
bitnami is part of VMware and is very wildly used and trusted so I think it's ok to use it

Reply:
Bitami is part of VMware. I didn't know it for so many years...